theano/gof/graph.py

```diff
@@ -822,7 +822,7 @@ def clone(i, o, copy_inputs=True):
     return [equiv[input] for input in i], [equiv[output] for output in o]
 
 
-def clone_get_equiv(inputs, outputs, copy_inputs_and_orphans=True, memo=None):
+def clone_get_equiv(inputs, outputs, copy_inputs=True, copy_orphans=True, memo=None):
```

Comment:
I get what you want but it's probably safer to make a new function (maybe clone2) that does what you want and call that.

Because otherwise we may end up chasing a weird bug in six months due to that change.